modules/installation-configuration-parameters.adoc

```diff
@@ -1284,6 +1283,52 @@ Optional VMware vSphere machine pool configuration parameters are described in t
 
 endif::vsphere,vmc[]
 
+ifdef::ash-default[]
+[id="installation-configuration-parameters-additional-azure-stack-hub_{context}"]
+== Additional Azure Stack Hub configuration parameters
+
+Additional Azure configuration parameters are described in the following table:
+
+.Additional Azure Stack Hub parameters
+[cols=".^2,.^3a,.^3a",options="header"]
+|====
+|Parameter|Description|Values
+
+|`platform.azure.armEndpoint`
+|The URL of the Azure Resource Manager endpoint that your Azure Stack Hub operator provides.
```

Comment:
```suggestion
|The URL of the Azure Resource Manager endpoint that your Azure Stack Hub Operator provides.
```
(unless this isn't that kind of Operator?)

Reply:
Correct. Not that kind of operator.